core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -635,6 +635,67 @@ public static function insertTagToSrc($data)
 		return $return;
 	}
 
+	/**
+	 * Adds {{env::path}} to relative links
+	 *
+	 * @param string $data The markup string
+	 *
+	 * @return string
+	 */
+	public static function addEnvPath($data)
+	{
+		$return = '';
+		$paths = preg_split('/((src|href)="([^"]+)")/i', $data, -1, PREG_SPLIT_DELIM_CAPTURE);
+
+		for ($i=0, $c=\count($paths); $i<$c; $i+=4)
+		{
+			$return .= $paths[$i];
+
+			if (!isset($paths[$i+1]))
+			{
+				continue;
+			}
+
+			if (preg_match('(^(https?://|#|/|\{\{))i', $paths[$i+3]))
+			{
+				$return .= $paths[$i+2] . '="' . $paths[$i+3] . '"';
+			}
+			else
+			{
+				$return .= $paths[$i+2] . '="{{env::path}}' . $paths[$i+3] . '"';
```

Comment:
```suggestion
				$return .= $paths[$i+2] . '="{{env::base_path}}/' . $paths[$i+3] . '"';
```
The prefixes need to be changed then everywhere to this 😬

Reply:
Changed in 56aa6a53b57d6db22ab782ffac95d8a2e5d2effe.